content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -431,6 +431,14 @@ Used on: PersistentVolumeClaim
 
 This annotation has been deprecated.
 
+### volume.beta.kubernetes.io/storage-class (deprecated)
+
+Used on: PersistentVolume, PersistentVolumeClaim
+
+This annotation can be used for PersistentVolume(PV) or PersistentVolumeClaim(PVC) to specify the name of [StorageClass](/docs/concepts/storage/storage-classes/).
```

Comment:
1. What happens if you also specify an actual `storageClassName`?

We should aim to document that. Even so, this is a good start. I'm happy to merge it.

Reply:
@sftim Thanks for the feedback. I have now added additional info as requested. Could you please review the latest changes again and let me know any feedback for it.